org.gnome.Contacts.yml

```diff
@@ -119,6 +126,17 @@ modules:
         url: https://download.gnome.org/sources/folks/0.14/folks-0.14.0.tar.xz
         sha256: d6e778da637d3b33fd9a6b45115e8157d8868211dc08b943d1075b5eb7282bb9
 
+  - name: libhandy
+    buildsystem: meson
+    config-opts:
+      - -Dexamples=false
+      - -Dglade_catalog=disabled
+      - -Dtests=false
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libhandy/-/archive/1.0.0/libhandy-1.0.0.tar.gz
```

Comment:
Use download.gnome.org whenever possible

Reply:
Any specific reason? Are those files cached?

Reply:
Done, although I still wonder why

Reply:
Because download.gnome.org has and will always be there :)